world/config/skillmanager.py

```diff
@@ -0,0 +1,69 @@
+from world.abilities.ability import Ability
```

Comment:
I would prefer a flatter directory structure here, so delete config and put all abilities in world/abilities. All spells can go in world/spells.